ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/ConnectorList.java

```diff
@@ -39,6 +40,7 @@ public ConnectorList(
     this.connectors = ImmutableList.copyOf(Objects.requireNonNull(connectors, "connectors"));
   }
 
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP", justification = "connectors is ImmutableList")
   public ImmutableList<SimpleConnectorInfo> getConnectors() {
```

Comment:
Compare https://github.com/confluentinc/ksql/pull/7809/files#r671807853

Or should we change the return type to `List`?